src/mame/motorola/mex68kecb.cpp

```diff
@@ -0,0 +1,339 @@
+// license:BSD-3-Clause
+// copyright-holders:Chris Hanson
+/*
+ * mex68kecb.cpp - Motorola MEX68KECB
+ *
+ * Created on: August 31, 2024
+ *     Author: Chris Hanson
+ *
+ * Documentation:
+ *   http://www.bitsavers.org/components/motorola/68000/MEX68KECB/MEX68KECB_D2_EduCompBd_Jul82.pdf
+ *
+ * The Motorola MC68000 Educational Computer Board is a single-board computer with
+ * a 4MHz 68000 CPU, 32KB RAM, 16KB ROM, host and terminal serial ports, a
+ * parallel interface/timer, a cassette interface, and a prototyping area with
+ * full access to the 68000 bus. The ROM contains TUTOR, a debug and bootstrap
+ * system that was the predecessor of MACSBUG.
+ *
+ * Specifications:
+ * - 4MHz MC68000L4 CPU
+ * - MC6850 ACIA x 2
+ * - MC68230 PIT
+ *
+ * To Do:
+ * - Cassette I/O
+ * - Save/Restore
+ *
+ */
+
+#include "emu.h"
+
+#include "bus/rs232/rs232.h"
+#include "cpu/m68000/m68000.h"
+#include "machine/68230pit.h"
+#include "machine/6850acia.h"
+#include "machine/mc14411.h"
+
+
+#include "debugger.h"
+
+#define VERBOSE  1
```

Comment:
'1' for VERBOSE is LOG_GENERAL.  But also things normally shouldn't be checked in with logging enabled.

Also, I don't see any LOG() or LOGMASKED() usage in this file?

Reply:
Yeah, I used some logging while I brought it up just to ensure things were what I expected, but it wasn't needed once I was done. Think I should delete the whole section?